bundler/doc/development/PULL_REQUESTS.md

```diff
@@ -10,7 +10,7 @@ Before you submit a pull request, please remember to do the following:
 
 Make sure the code formatting and styling adheres to the guidelines. We use RuboCop for this. Lack of formatting adherence will result in automatic GitHub Actions build failures.
 
-      $ bin/rubocop -a
+      $ rake 'rubocop:bundler[-a]'
```

Comment:
This documentation file assumes you're in the bundler folder, so the new version will no longer work. Since we have now provided a working `bin/rubocop` binstub, we can revert this change.

Reply:
Dropped the change.